client/state/plugins/reducer.ts

```diff
@@ -14,20 +13,13 @@ import premium from './premium/reducer';
 import installed from './installed/reducer';
 import upload from './upload/reducer';
 import recommended from './recommended/reducer';
-import marketplace from './marketplace/reducer';
-import { IPurchaseFlowState } from './marketplace/types';
-
-export interface IAppState extends DefaultRootState {
```

Comment:
Since this is mean to be common to all state slices, it is moved to the `state/types.ts` file